aloscene/bounding_boxes_3d.py

```diff
@@ -500,7 +500,30 @@ def get_view(self, frame, size: Union[tuple, None] = None, mode: str = "3D", **k
             )
             # Draw bbox 3d
             frame = frame.norm01().cpu().rename(None).permute([1, 2, 0]).detach().contiguous().numpy()
-            draw_3D_box(frame, vertices_3d_proj)
+            if isinstance(self.labels, dict):
+                labels_list = []
+                # Sorting the labels set to always have the same order
+                for label_key in sorted(self.labels.keys()):
+                    labels_list.append(self.labels[label_key])
+                class_names = []
+                color = []
+                for index in range(len(labels_list[0])):
+                    text = []
+                    for label in labels_list:
+                        text.append(
+                            label.labels_names[int(label[index])] if label.labels_names else str(int(label[index]))
+                        )
+                    class_names.append(" , ".join(text))
+                    labels_sum = sum([int(label[index]) for label in labels_list])
+                    color.append(self._GLOBAL_COLOR_SET[labels_sum % len(self._GLOBAL_COLOR_SET)])
+                labels = range(len(class_names))
+            else:
+                labels = [int(label_value) for label_value in self.labels]
```

Comment:
is this case you need to put back the values of labels to the good range, otherwise an out of index error is raised. You can reproduce this error by running your code [here](https://github.com/Visual-Behavior/aloception-oss/issues/253)